cmd/sriovdp/sriov-device-plugin.go

```diff
@@ -80,37 +75,11 @@ func newSriovManager() *sriovManager {
 		glog.Errorf("Error. Could not create K8s Client using supplied config. %v", err)
 		return nil
 	}
-	sm := &sriovManager{
+	return &sriovManager{
 		k8ClientSet:      clientset,
```

Comment:
K8sClientSet no longer needed if there's no communication to the API server. You can remove this as well